recipes/svdb/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "1.0.7" %}
+{% set md5sum = "2764c03866f3aa827bec3fcc50fa1742" %}
+
+package:
+  name: svdb
+  version: "{{ version }}"
+
+source:
+  fn: svdb-{{ version }}.tar.gz
+  url: https://github.com/J35P312/SVDB/archive/SVDB-{{ version }}.tar.gz
+  md5: "{{ md5sum }}"
+
+build:
+  skip: True  # [not py27]
+  number: 0
```

Comment:
Can you add
```yaml
  entry_points:
    - svdb = svdb.__main__:main
```
to the build section.

Reply:
Argh, I wasn't notified of your update so I didn't review them/approve them (maybe GitHub doesn't notify if no commit was added and force pushed -- strange thing...). That `entry_points:` should've been part of the build section.

Reply:
Gosh, ok, let's continue this on #7565?